src/ResourceManager/Websites/Commands.Websites/help/Swap-AzureRmWebAppSlot.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Please remove this.  I think this is a vestige of when there used to be such a cmdlet